Sources/TokamakCore/Fiber/FiberReconciler.swift

```diff
@@ -39,6 +39,7 @@ public final class FiberReconciler<Renderer: FiberRenderer> {
   private let caches: Caches
 
   private var isReconciling = false
+  private var changedFibers = Set<ObjectIdentifier>()
```

Comment:
nit: probably worth adding a doc comment explaining why these need to be tracked in a `Set` and they're used throughout the reconciler